com.microsoft.mrtk.input/Utilities/EyeCalibrationChecker.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices.WindowsRuntime;
+using UnityEngine;
+using UnityEngine.Events;
+#if WINDOWS_UWP
+using Windows.Perception;
+using Windows.Perception.People;
+using Windows.Perception.Spatial;
+using Windows.UI.Input.Spatial;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// A helper class used to check eye calibration status. Only for UWP Platforms. 
+    /// </summary>
+    [AddComponentMenu("MRTK/Input/Eye Calibration Checker")]
+    public class EyeCalibrationChecker : MonoBehaviour
+    {
+        #region Public Enums
+
+        public enum EyeCalibrationStatus
+        {
+            /// <summary>
+            /// The eye calibration status is not defined. 
+            /// </summary>
+            Undefined,
+
+            /// <summary>
+            /// The eye calibration status could not be retrieved because this is a non-UWP device.
+            /// </summary>
+            IsNonUWPDevice,
```

Comment:
Let's remove UWP from this enum value.   Later platforms maybe supported later, and I wouldn't want to "bake-in" the word UWP because of that.

Can you rename to something like `Unsupported`  or `StatusUnsupported`